starlite/app.py

```diff
@@ -283,6 +286,7 @@ def __init__(
             on_shutdown=on_shutdown or [],
             on_startup=on_startup or [],
             openapi_config=openapi_config,
+            opt=opt,
```

Comment:
```suggestion
            opt=opt or {},
```

Reply:
Can the optional be discarded for the config object like the other optional collections?